Source/Editor/Modules/WindowsModule.cs

```diff
@@ -709,7 +709,7 @@ public override void OnInit()
             var dpiScale = Platform.DpiScale;
             var settings = CreateWindowSettings.Default;
             settings.Title = "Flax Editor";
-            settings.Size = new Vector2(1300 * dpiScale, 900 * dpiScale);
+            settings.Size = new Vector2(1300 * dpiScale, 900 * dpiScale); // TODO: Place the window on the correct screen (and use that screen's size)
```

Comment:
What would a decent implementation of this look like? Currently it makes the main window too large/small on my screens on startup.

Reply:
The best would be the center of the main display. Is there a way to query DPI of the primary monitor?

Reply:
Okay, will do. 
And yes, `Platform.DpiScale` currently returns the DPI of the primary monitor. I'll add that to the documentation comment there

Reply:
Hooray, I managed to fix that and that super irritating bug where the window has the wrong size when starting!